v2/support/instances_sample_app/assets/iframe.html

```diff
@@ -0,0 +1,37 @@
+<html>
+<body>
+  <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
+  <script>
+    // Initialise the Zendesk JavaScript API client
+    var client = ZAFClient.init();
+
+    function commentTemplate(currentUser, requester, lineBreak) {
+      return 'Hi ' + currentUser + ',' + lineBreak + lineBreak + 'Kind regards,' + lineBreak + requester;
+    }
+
+    // instance.created fires whenever an instance of your app is created
+    client.on('instance.created', function(context) {
+      // handle all ticket_sidebar instances
+      if (context.location === 'ticket_sidebar') {
```

Comment:
I was going to, but then I get an error on `get('ticket.requester.name')`, I'd also probably need to handle the template differently in case there's no requester.